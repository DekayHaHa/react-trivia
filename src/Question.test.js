// what do we need to import?


// what kind of mock data might we need to create for ourselves?

describe('Question', () => {

  // what kind of setup do we need to do for our tests?


  it('should match the snapshot with all data passed in correctly', () => {
    // how do we test our rendered output?
  });

});



