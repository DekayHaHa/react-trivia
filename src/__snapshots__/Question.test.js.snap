// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question should match the snapshot with all data passed in correctly 1`] = `
<li
  onClick={[Function]}
>
  <p>
     + 
    When was the first Call of Duty title released?
  </p>
</li>
`;
